tests/ui/parallel-rustc/cycle_crash.rs

```diff
@@ -0,0 +1,5 @@
+//@ compile-flags: -Z threads=2
+//@ check-fail
+
+const FOO: usize = FOO; //~ERROR `main` function not found in crate `cycle_crash
```

Comment:
I think we can add a `main` function just to remove this error and focus on the cycle fatal issue.

Reply:
Done